docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -78,7 +77,7 @@ C:\Program Files\LLVM\bin\ld.lld.exe
 
 OCaml
 ---------------------------------
-Install [OCaml for Windows (64-bit)](https://www.ocamlpro.com/pub/ocpwin/ocpwin-builds/ocpwin64/20160113/).
+Install [OCaml for Windows (64-bit)](https://www.ocamlpro.com/pub/ocpwin/ocpwin-builds/ocpwin64/20160113/ocpwin64-20160113-4.01.0+ocp1-mingw64.exe).
```

Comment:
We did not want to add the path to the exe directory i.e. this was intentional. So, please do not make this change.

Reply:
The problem I had with the previous link is that when I got there, I don't know exactly which file to download. All the file names are truncated to a point that they all look the same. I had to try one by one until I found the third file in the list is the desired one. What's the intention behind giving out the directory instead of the file?

Reply:
I was not allowed to point directly to the exe path for various reasons, all of which still hold
1) The version of ocaml is updated and so our link can get obsolete very soon.
2) The exe gets downloaded and starts installing automatically.

Yes, the opening page is not displayed properly. But, if you hover on the link, you will see the complete path displayed at the bottom of your browser window.

How about? Displaying the complete path but not making it clickable. Cut and paste will work if the version of ocaml still exists.

Reply:
Thanks for  the suggestion.